cuBERT_topic_modelling/tests/test_umap_dr.py

```diff
@@ -0,0 +1,30 @@
+import sys
+from os.path import dirname, realpath
+import numpy as np
+from sklearn.datasets import fetch_20newsgroups
+import pytest
+
+filepath = realpath(__file__)
+dir_of_file = dirname(filepath)
+parent_dir_of_file = dirname(dir_of_file)
+parents_parent_dir_of_file = dirname(parent_dir_of_file)
+sys.path.append(parents_parent_dir_of_file + "/cuBERT-topic-modelling/")
```

Comment:
We should get ride of `sys.path.Hacks` . I think we should add a setup.py to run  these tests. See below for reference on how to set it up. 

https://stackoverflow.com/a/50193944/7904797